CMakeLists.txt

```diff
@@ -460,17 +484,18 @@ if (BUILD_TESTING)
     endif()
 
     if (S2N_INTERN_LIBCRYPTO)
-        # if libcrypto was interned, rewrite libcrypto symbols so use of internal functions will link correctly
+        # if libcrypto was interned, rewrite libcrypto symbols so use of internal
+        # functions will link correctly
         add_custom_command(
             TARGET testss2n POST_BUILD
             COMMAND
                 objcopy --redefine-syms libcrypto.symbols lib/libtestss2n.a
         )
     endif()
 
-    #run unit tests
-    file (GLOB TEST_LD_PRELOAD "tests/LD_PRELOAD/*.c")
```

Comment:
Removing this because it is unused. I searched the entire code (not just CMakeLists.txt) and found that this library is only used when unit tests are run through make. See the PR overview for my thoughts on why we should totally delete this library.